core/db/migrate/20150806190833_add_id_and_timestamp_to_promotion_rule_user.rb

```diff
@@ -0,0 +1,13 @@
+class AddIdAndTimestampToPromotionRuleUser < ActiveRecord::Migration
+  def up
+    add_column :spree_promotion_rules_users, :id, :primary_key
+    add_column :spree_promotion_rules_users, :created_at, :datetime
+    add_column :spree_promotion_rules_users, :updated_at, :datetime
+  end
+
+  def down
+    remove_column :spree_promotion_rules_users, :updated_at
+    remove_column :spree_promotion_rules_users, :created_at
+    remove_column :spree_promotion_rules_users, :id
+  end
```

Comment:
I'm not sure if I'm :+1: on this or :-1: on this.

After some thought I think we should add these now for "join" tables and if developers prefer to not "clutter" their join tables with timestamps and/or primary keys, then they can remove it themselves.